go/teams/service_helper.go

```diff
@@ -661,20 +661,38 @@ func AcceptInvite(ctx context.Context, g *libkb.GlobalContext, token string) err
 	return err
 }
 
-func ParseAndAcceptSeitanToken(ctx context.Context, g *libkb.GlobalContext, tok string) (wasSeitan bool, err error) {
+func parseAndAcceptSeitanTokenV1(ctx context.Context, g *libkb.GlobalContext, tok string) (wasSeitan bool, err error) {
```

Comment:
we probably also want to rename `GenerateIKeyFromString` to `ParseIKeyFromString` here too

Reply:
done